components/FoodFullResults.js

```diff
@@ -56,13 +55,29 @@ const ResultsTable = props => {
               </TableRow>
             </TableHead>
             <TableBody>
-              {values.map((value, vIndex) => (
-                <TableRow key={vIndex}>
-                  {Object.keys(value).map((key, kIndex) => (
-                    <TableCell key={kIndex} className={classes.cell}>
-                      {value[key]}
-                    </TableCell>
-                  ))}
+              {values.map(({ food }, key) => (
+                <TableRow key={key}>
+                  <TableCell className={classes.foodName}>
+                    {food.label}
+                  </TableCell>
+                  <TableCell className={classes.cell}>
+                    {food.nutrients.ENERC_KCAL}
```

Comment:
does Edamam API guarantee `nutrient` value? if not then this will crash app. if `ENERC_KCAL` is missing then we have ugly undefined here.

Reply:
Yep. This is a little bit safer:
```suggestion
                    {nutrients && food.nutrients.ENERC_KCAL}
```

Reply:
I think it is always returned, but it's better to make it safer.